src/AppInstallerCLICore/Workflows/WorkflowBase.h

```diff
@@ -127,6 +127,12 @@ namespace AppInstaller::CLI::Workflow
     // Inputs: None
     // Outputs: None
     void EnsureRunningAsAdmin(Execution::Context& context);
+
+    // Ensures that the feature is enabled.
+    // Required Args: None
+    // Inputs: Feature
+    // Outputs: None
+    void EnsureFeatureEnabled(Execution::Context& context);

```

Comment:
>EnsureFeatureEnabled [](start = 9, length = 20)

Don't read the feature out of a context value, construct an object (see VerifyFile above).